.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: c
-sudo: required
+sudo: false
+services:
+  - docker
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
 script: bash -ex .travis-opam.sh
 env:
-  - OCAML_VERSION=4.02 PACKAGE=enumerators
-  - OCAML_VERSION=4.03 PACKAGE=enumerators
-  - OCAML_VERSION=4.04 PACKAGE=enumerators
-os:
-  - linux
+  global:
+    - PACKAGE=enumerators
+    - DISTRO=apline
```

Comment:
I don't know this distribution :)

Reply:
I love Alpine!  Check the spelling.  The CI passed nonetheless, which sounds fishy.

Reply:
uh! Indeed!